plugins/experimental/stale_while_revalidate/README

```diff
@@ -1,3 +1,13 @@
+************************************************************
+    THIS PLUGIN IS DEPRECATED IN FAVOR OF CHANGE TO CORE
+************************************************************
+
+You can still run `make && make install` in this directory
```

Comment:
It will since it's still in `configure.ac`. It just won't build with a top level `make`.